doc/source/tutorials/Deembedding.ipynb

```diff
@@ -167,7 +167,7 @@
    "source": [
     "## Through-only de-embedding\n",
     "\n",
-    "At higher frequencies above 10 GHz, the fringe capacitance of the open dummy and the parasitic inductance of the short dummy cause errors. For high frequencies, several methods using a through (thru) dummy have been proposed, such as `SplitPi`[^1], `SplitTee`[^2], `AdmittanceCancel`[^3], and `ImpedanceCancel`[^4]. The thru dummy is a direct connection of the left and right test pads. The example of the test device and thru dummy is shown in the figure below, where G and S indicate ground pads and signal pads.\n",
+    "At higher frequencies above 10 GHz, the fringe capacitance of the open dummy and the parasitic inductance of the short dummy cause errors. For high frequencies, several methods using a through (thru) dummy have been proposed, such as `SplitPi`[[1](#ref1)] , `SplitTee`[[2](#ref2)]_ , `AdmittanceCancel`[[3](#ref3)], and `ImpedanceCancel`[[4](#ref4)]. The thru dummy is a direct connection of the left and right test pads. The example of the test device and thru dummy is shown in the figure below, where G and S indicate ground pads and signal pads.\n",
```

Comment:
Thank you for fixing the references.
There's an unnecessary underscore.
`SplitTee`[[2](#ref2)]_